### PR TITLE
Helptext Feature

### PR DESCRIPTION
**Pull request by MalteMagnussen**
### Related Taiga: 
* [Task](https://tree.taiga.io/project/maltemagnussen-fog/task/193?kanban-status=1592820)
* [Task](https://tree.taiga.io/project/maltemagnussen-fog/task/184?kanban-status=1592823)

### CODE REVIEW

File: FogCarport/src/main/java/presentation/viewOrder.java [hunk 1]
```diff
@@ -34,9 +34,9 @@ String execute(HttpServletRequest request, LogicFacade logic) throws LoginExcept
...
         // Place values used by viewOrder on request.
         session.setAttribute("order", order);
-        session.setAttribute("tile", logic.getMaterial(order.getRoof_tiles_id()).getDescription());
-        session.setAttribute("shedwalls", logic.getMaterial(order.getShed_walls_id()).getDescription());
-        session.setAttribute("shedfloor", logic.getMaterial(order.getShed_floor_id()).getDescription());
+        session.setAttribute("tile", logic.getMaterial(order.getRoof_tiles_id(), "roof").getDescription());
+        session.setAttribute("shedwalls", logic.getMaterial(order.getShed_walls_id(), "shed").getDescription());
+        session.setAttribute("shedfloor", logic.getMaterial(order.getShed_floor_id(), "shed").getDescription());
         session.setAttribute("customer", logic.getCustomer(order.getId_customer()));
         session.setAttribute("employee", logic.getEmployee(order.getId_employee()));
```
Comment (Castau) [inline FogCarport/src/main/java/presentation/viewOrder.java]: i think this part is going to create a conflict with the fix of session-use.

Comment (MalteMagnussen) [reply FogCarport/src/main/java/presentation/viewOrder.java]: ah fuck, true
Is the session-use fix on developer? Then I can fix it in a developer -> Helptext merge

Comment (Castau) [reply FogCarport/src/main/java/presentation/viewOrder.java]: Den ligger som pull-request til dev

Comment (MalteMagnussen) [reply FogCarport/src/main/java/presentation/viewOrder.java]: Fixed.

File: FogCarport/src/main/java/presentation/payOrder.java [hunk 2]
```diff
@@ -35,9 +35,9 @@ String execute(HttpServletRequest request, LogicFacade logic) throws LoginExcept
...
         // Place values used by viewOrder on request.
         session.setAttribute("order", order);
-        session.setAttribute("tile", logic.getMaterial(order.getRoof_tiles_id()).getDescription());
-        session.setAttribute("shedwalls", logic.getMaterial(order.getShed_walls_id()).getDescription());
-        session.setAttribute("shedfloor", logic.getMaterial(order.getShed_floor_id()).getDescription());
+        session.setAttribute("tile", logic.getMaterial(order.getRoof_tiles_id(), "roof").getDescription());
+        session.setAttribute("shedwalls", logic.getMaterial(order.getShed_walls_id(), "shed").getDescription());
+        session.setAttribute("shedfloor", logic.getMaterial(order.getShed_floor_id(), "shed").getDescription());
         session.setAttribute("customer", logic.getCustomer(order.getId_customer()));
         session.setAttribute("employee", logic.getEmployee(order.getId_employee()));
```
Comment (Castau) [inline FogCarport/src/main/java/presentation/payOrder.java]: shouldn't session use here be changed to request - i think something might have gone wrong in a merge somewhere

Comment (MalteMagnussen) [reply FogCarport/src/main/java/presentation/payOrder.java]: It is @HrBjarup  method. Not quite sure if its right or not. There was no merge conflict with that class.

**Comment by MalteMagnussen** [comment]
image
It works.

**Pull request merged**